radio/src/pulses/afhds3.cpp

```diff
@@ -0,0 +1,661 @@
+/*
```

Comment:
cosmetics: is it possible you check the indentation, new lines, spaces in this file?

Reply:
Reformatted using settings: 
![obraz](https://user-images.githubusercontent.com/3341647/82154400-e3449000-986d-11ea-9ebb-632dc808a492.png)
Feel free to edit attached to this PR Eclipse style. Maybe we can publish it later as official style for the project?

Reply:
We have included a CLion style sheet in the repo, but sure, we could add another ide too

Reply:
@3djc can you just list all settings?
I don't want to install next IDE to get those settings.
Eclipse and QT editor settings would make sense to me. CLion is not free software so it is not fitting in project at all.

Reply:
It is free for open source projects :)

Reply:
This code style is not the one we use in opentx. 
For example if / else if / else is completely wrong